css/src/components/layout.scss

```diff
@@ -214,7 +214,7 @@ $default-flyout-width-widescreen: 480px;
 					#{$layout-flyout-width-name}
 				);
 			grid-template-areas:
-				'header header flyout'
+				'header header header'
 				'hero hero flyout'
 				'menu main flyout'
 				'footer footer flyout';
```

Comment:
why do you have 'footer footer flyout' here in line 220 but 'footer footer footer footer' in line 169? like, is line 220 supposed to say 'footer footer footer'?

Reply:
Great catch! Updated